ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommand.java

```diff
@@ -48,7 +48,7 @@
 public class HashicorpSubCommand extends SignerSubCommand {
   static final String COMMAND_NAME = "hashicorp-signer";
   private static final String DEFAULT_HASHICORP_VAULT_HOST = "localhost";
-  private static final String DEFAULT_KEY_PATH = "/secret/data/ethsignerSigningKey";
+  private static final String DEFAULT_KEY_PATH = "/v1/secret/data/ethsignerSigningKey";
```

Comment:
Probably should have a unit test for this

Reply:
this is true - however, need it in.